common/religions/necromancy_group.txt

```diff
@@ -135,4 +135,59 @@ necromancy_group = {
 
 		dynamic_cult = no	# Warcraft - can't handle it yet
 	}	
+	throsic = {
+		graphical_culture = westerngfx
+
+		icon = 41
+		heresy_icon = 42
+		
+		color = { 5 105 120 }
+		
+		scripture_name = NECRONOMICON
+		priest_title = NECROMANCER
+		piety_name = IMPIETY
+		
+		high_god_name = THROS
+		
+		god_names = {
+			THROS THE_BLIGHT THE_NECROMANCY
+		}
+		evil_god_names = {
+			THE_LIGHT THE_LIFE
+		}
+		
+		# Modifiers
+		character_modifier = {
+			general_opinion = -25
+			same_religion_opinion = 25
+			rel_head_opinion = 25
+			burning_legion_religion_opinion = -175
+			death_god_opinion = 25
+			old_gods_worship_opinion = -150
+			
+			# Must be in every religion
+			uncolonized_trait_opinion = -200
```

Comment:
Needs rework. `uncolonized_trait_opinion` not used anymore. All religions have small bonuses and maluses (the list can be found [Here](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/wiki/Links-to-External-Documentation). Order of modifiers are changed.